lib.py

```diff
@@ -557,44 +598,14 @@ def increment_outlays(self, amount_msat):
 
         self.save()
 
-    def update_outlays(self, payment_results):
-        new_outlays = {}
-        for member_id, outlay in self.outlays.items():
-            payment_amount = 0
-            payment_result = payment_results.get(member_id, None)
-
-            if payment_result:
-                status = payment_result["status"]
-                if status != "complete":
-                    self._plugin.log(f"Failed to pay member {member_id}")
-                    new_outlays[member_id] = outlay
-                    continue
-            
-                payment_amount = payment_result.get("amount_sent_msat", 0)
-            else:
-                self._plugin.log(f"No payment_result for member {member_id}. This could indicate a failed payment. Outlays will remain unchanged.", "warn")
-
-            new_outlay = Millisatoshi(outlay) - Millisatoshi(payment_amount)
-
-            new_outlays[member_id] = new_outlay
-
-        self._plugin.log(f"New outlay values after decrementing: {new_outlays}")
-        self.outlays = new_outlays
-
-        self.save()
-
     def pay(self, amount_msat):
 
         payment_results = None
         self._plugin.log(f"Calculating total outlays...")
         total_outlays = amount_msat * self.prism.outlay_factor
-        self._plugin.log(f"Total outlays are {total_outlays} after applying an outlay factor of {self.prism.outlay_factor} to the income amount {amount_msat}.")
+        self._plugin.log(f"Total outlays will be {total_outlays} after applying an outlay factor of {self.prism.outlay_factor} to the income amount {amount_msat}.")
         self.increment_outlays(amount_msat=total_outlays)
-
```

Comment:
This TODO item was in there a while. This needed to be done. It reared it's head sooner than expected.